config.yaml

```diff
@@ -39,10 +39,10 @@
             # Make sure that the samples you merge have the same sample features (e.g paired, sense, dup_type)
             # Choose an informative experiment name
   experiment_name: {
-                    rbp: 'rbp name', #rna binding protein name, eg: PUM2,
-                    replicates: ["fg_sample1"], # sample(s) to be treated as foreground.
+                    rbp: 'PUM2', #rna binding protein name, eg: PUM2,
+                    replicates: ["ENCFF041KJT_ENCFF462SCV"], # sample(s) to be treated as foreground.
```

Comment:
I presume these are actual file names. But the extension (.bam/something else) is not included. So here we need more explanation for what these fields represent